Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM ubuntu:16.04
 
 RUN apt-get update
-RUN apt-get install -y python3-pip
+RUN apt-get install -y wget curl python3-pip
```

Comment:
FYI: I found using a python version manager, such as conda, pyenv,  is more convenient.  For example, we are using this approach in tf-plus: https://github.com/tensorflow/io/blob/master/dev/Dockerfile#L19

Reply:
@zou000 Thanks for the suggestion. Adding `source activate tfio-dev` to `~/.bashrc` neatly separates the Linux built-in Python2 and our Python3. This approach avoids the error-prone
```bash
# Make python3 as default Python interpreter
RUN ln -s /usr/bin/python3 /usr/bin/python
RUN chmod +x /usr/bin/python
```

Reply:
Update: enabling python env in `.bashrc` has certain drawbacks. For example, if you do `docker run image_name bash test.sh` i.e. non-interactive bash, `.bashrc` is skipped, so the python env is not activated. One workaround is to define the `source activate` in `entrypoint`.

Reply:
I can think of 2 other ways:

1. Invoke bash interactively e.g. ` docker run image_name bash -i test.sh`
2. The idea of a pyenv is very simple, it just needs to find the python binary under the env root dir, e.g. in your case, you just need to modify PATH:
```bash
ENV PATH=/miniconda/envs/sqlflow-dev/bin:$PATH
```  
I like the PATH approach better. You can verify the result by:
```
docker run image_name python -c 'import sys; print(sys.path)'

```